### ISSUE TITLE
[Bug] Since 2.78 docfx fails if node.js is not found even when not generating pdf

### ISSUE THREAD

**Issue by kschelfthoutelement** [dotnet/docfx#10415]
**Describe the bug**
I understand node.js was removed from the DocFx package since v2.78. I also understand from the documentation that node.js should only be necessary when generating pdf.

Yet, I set `"pdf": false` in my config and I still get the exception. (I do indeed not have node.js installed)

**To Reproduce**
1. Part of config:

```json
{
  "build": {
    "globalMetadata": {
      "pdf": false
    }
  }
}
```

2. `dotnet tool run docfx docfx.json`

Exception complaing about node.js

Also tried adding `pdf: false` to my toc, didn't make any difference.

**Expected behavior**

Docfx generates the html

**Context (please complete the following information):**
- OS: Windows/docker
- Docfx version: 2.78

**Additional context**

**Comment by filzrev** [comment]
Reported issue is resolved by #10406.
But it's not released yet.

**Comment by kschelfthoutelement** [comment]
Great! Sorry I missed that. Thanks for the quick turnaround.
